src/components/common/applicant-info-nav.vue

```diff
@@ -1,16 +1,15 @@
 <template>
   <v-col cols="5" class="text-right py-0">
     <v-btn x-large
-           id="submit-back-btn"
-           class="mr-3"
+           class="submit-back-btn mr-3"
```

Comment:
I changed id to a classname (and also below) to get rid of "duplicate id" messages.

This is actually the only place where this id was used, but apparently NR UI sometimes renders 2 of these components, so you would get a duplicate id.